### PR TITLE
Edit UG formating

### PR DESCRIPTION
**Pull request by tayruxin**
- Removed table for the different styles of text in "Navigating the user guide" 
- Edited a typo error in quick tour

### CODE REVIEW

**Review by papataco14 — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -70,17 +70,14 @@ throughout this user guide.
 * Alerts you of potential pitfalls or things to be cautious of when using LinkMeIn.
 </div>
 
-Throughout this guide, you will also see **different styles of text**. Here are what each of them mean:
+Throughout this guide, you will also notice **various text styles**. Here's what each of them means:
+- _Represents possible scenarios that you can use a feature for (mainly in the feature sections)._
+- [Represents hyperlinks that you can click into, which will direct you to another section of this user guide or to a website.]():
```
Comment (GlendaChong) [inline docs/UserGuide.md:75]: Remove the semi-colon at the end.

**Review by GlendaChong — APPROVED** [review]
LGTM, but minor change required.

**Review by alientian — APPROVED** [review]
lgtm

**Pull request merged**